packages/paste-website/src/pages/patterns/filter-group/index.mdx

```diff
@@ -109,20 +110,6 @@ export const pageQuery = graphql`
       </Heading>
     </Card>
   </Column>
-  <Column>
```

Comment:
Note: ended up removing Stack and Flex from the list of ingredients because I didn't use them in the examples 👀